LandSandBoat/server
# [Rewrite] Convert Enfeebling spells to new system

**_I affirm:_**

## What does this pull request do?

 - Removes logic from enfeebling spells (White, Black and Ninjutsu) and uses global file to handle parameters and calculations.
 - Makes enfeebling spells return magic burst! message.
 - Makes STATUSRES modifiers work as they should, not affecting MEVA, but allowing for a separate check for spell nulification.
 - Makes potency resistance separate from duration resistance.
 - Added check for mob status immunities.

## Steps to test these changes

Cast ninjutsu, black and white enfeebling spells. Make sure server doesnt return any errors. Make sure the spells are aplied, or rsisted, etc...
